lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -994,174 +990,198 @@ def cmd_load_tabs(str, words)
 
   def cmd_route_help
     print_line "Route traffic destined to a given subnet through a supplied session."
+    print @@route_opts.usage
     print_line
     print_line "Usage:"
-    print_line "  route [add/remove] subnet netmask [comm/sid]"
-    print_line "  route [add/remove] cidr [comm/sid]"
-    print_line "  route [get] <host or network>"
-    print_line "  route [flush]"
-    print_line "  route [print]"
-    print_line
-    print_line "Subcommands:"
-    print_line "  add - make a new route"
-    print_line "  remove - delete a route; 'del' is an alias"
-    print_line "  flush - remove all routes"
-    print_line "  get - display the route for a given target"
-    print_line "  print - show all active routes"
+    print_line "  route [--add/--remove] subnet netmask [comm/sid]"
```

Comment:
I'd be good to let this function still have its own custom parsing to help keep the usage of `verbs` 👀

Reply:
The `route` command can still take in the likes of `add`, `get`, `flush`. However they are normalised to `--add`, `--get` etc. Let me know if you would like this changed 👍

Reply:
I think the previous functionality worked well here, let's ctrl+z these parts 🎉 

Unless there's an easy way to support this functionality directly in rex parser to help support this use case? 🤔

Reply:
Sounds fair enough! I'm happy to undo it 👍 It would be possible, but I'm not sure what can of worms that would open when trying to parse words that aren't `-f` or `--flag` 🤔